apps/docs/utils/addContent.ts

```diff
@@ -8,19 +8,19 @@ export async function addContentToDb(
 	content: GeneratedContent
 ) {
 	const sectionInsert = await db.prepare(
-		`REPLACE INTO sections (id, idx, title, description, path, sidebar_behavior) VALUES (?, ?, ?, ?, ?, ?)`
+		`INSERT INTO sections (id, idx, title, description, path, sidebar_behavior) VALUES (?, ?, ?, ?, ?, ?)`
```

Comment:
Will these changes mean the db will have an error if the id is already there?

Reply:
yeah, but it clears the db before running so if there's something there already it's overwriting a different bit of content and is therefore an error we want to throw.